koku/masu/test/external/downloader/aws_local/test_aws_local_report_downloader.py

```diff
@@ -86,13 +87,14 @@ def tearDown(self):
         shutil.rmtree(DATA_DIR, ignore_errors=True)
         shutil.rmtree(self.fake_bucket_name)
 
+    @patch("masu.util.aws.common.remove_files_not_in_set_from_s3_bucket")
     @patch("masu.processor.parquet.parquet_report_processor.settings", ENABLE_S3_ARCHIVING=True)
-    def test_download_bucket(self, _):
+    def test_download_bucket(self, _, __):
```

Comment:
What's the purpose of defining a specific number of unused parameters? Prevent being called with too few or too many positional args?

If we don't care about the number of args, it reads better to use `*args`.
```suggestion
    def test_download_bucket(self, *args):
```

Reply:
👍🏿 to `*args`